### DISCUSSION TITLE
Allow Single Click to Open All Resolved Conversations OR Allow a Personal Settings to Always View them Opened<title>

### DISCUSSION THREAD

**Discussion by jeffsaremi** [community/community#47500] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

As a final reviewer /approver of a PR we want to be able to view every conversation in opened form even if they were previously resolved. This can be before or after a merge.

Could we have a single button that allows us to do that? Or can we have a personal setting that always opens PRs in a view that expands all resolved/unresolved converstions?

**Comment by stanleyEIQ** [comment]
💯 this is an incredibly annoying part of the github code review UI, along with this related issue - https://github.com/orgs/community/discussions/16111

**Comment by huyenltnguyen** [comment]
I just found that if we Alt + click / Option + click on any resolved conversation, all resolved conversations will be shown/expanded.

Hope this helps.
